python/lsst/ap/association/utils.py

```diff
@@ -124,13 +126,10 @@ def readSchemaFromApdb(apdb):
 
     Returns
     -------
-    schemaTable : 'dict' of `lsst.dax.apdb.apdbSchema.ApdbSchema`
+    schemaTable : dict[str, schema_model.Table | None]
```

Comment:
Would it make more sense to not make an entry instead of returning `None`? I assume that's for tables that aren't in a particular DB instance, but it would be good to document that explicitly.